content/_code-samples/quickstart/js/ripplex8-escrow.js

```diff
@@ -0,0 +1,240 @@
+// *******************************************************
+// ************* Add Seconds to Current Date *************
+// *******************************************************
+
+function addSeconds(numOfSeconds, date = new Date()) {
```

Comment:
Instead of using an `addSeconds` method with a magic number of 946684800, a more semantically clear option would be to use one of xrpl.js's built-in time-conversion methods. If you want to convert numeric seconds, [`unixTimeToRippleTime(...)`](https://js.xrpl.org/functions/unixTimeToRippleTime.html) is the one, or if you want to convert from a `Date` instance, [`isoTimeToRippleTime(...)` method](https://js.xrpl.org/functions/isoTimeToRippleTime.html) does that.

Tip: When it comes time to do the Python equivalent, xrpl-py has [`posix_to_ripple_time(...)`](https://xrpl-py.readthedocs.io/en/stable/source/xrpl.utils.html#xrpl.utils.posix_to_ripple_time) which takes int/float seconds as input, and one for converting from Python's native Datetime class type too.